deploy/router/cloud1.yaml

```diff
@@ -17,6 +17,9 @@ spec:
     role: interior
   interRouterListeners:
   - port: 55672
+  interRouterConnectors:
+  - port: 55672
+    host: cloud2.cloud2.svc.cluster.local
```

Comment:
cloud 2 should be connecting to cloud1, so there is no obvious value in cloud 1 connecting back... was something not working without this?

Reply:
Hm, I didn't have a full mesh. Maybe I didn't wait long enough. Let me retest

Reply:
In my mind I thought that there should be a full mesh between cloud. But you're right, there's no need for that